diff -rNU3 linux-6.1.11.old/arch/x86/Kconfig linux-6.1.11/arch/x86/Kconfig
--- linux-6.1.11.old/arch/x86/Kconfig	2023-02-09 11:28:28.000000000 +0100
+++ linux-6.1.11/arch/x86/Kconfig	2023-02-10 02:23:18.520145732 +0100
@@ -240,7 +240,7 @@
 	select HAVE_NOINSTR_HACK		if HAVE_OBJTOOL
 	select HAVE_NMI
 	select HAVE_NOINSTR_VALIDATION		if HAVE_OBJTOOL
-	select HAVE_OBJTOOL			if X86_64
+#	select HAVE_OBJTOOL			if X86_64
 	select HAVE_OPTPROBES
 	select HAVE_PCSPKR_PLATFORM
 	select HAVE_PERF_EVENTS
