--- util-linux-2.32.old/src/Makefile.in
+++ util-linux-2.32/src/Makefile.in
@@ -372,12 +372,12 @@
 @BUILD_SETTERM_TRUE@@HAVE_TINFO_FALSE@am__append_125 = $(NCURSES_CFLAGS)
 @BUILD_MESG_TRUE@am__append_126 = mesg
 @BUILD_MESG_TRUE@am__append_127 = term-utils/mesg.1
-@BUILD_WALL_TRUE@am__append_128 = wall
-@BUILD_WALL_TRUE@am__append_129 = term-utils/wall.1
-@BUILD_WALL_TRUE@@MAKEINSTALL_DO_CHOWN_TRUE@@USE_TTY_GROUP_TRUE@am__append_130 = install-exec-hook-wall
-@BUILD_WRITE_TRUE@am__append_131 = write
-@BUILD_WRITE_TRUE@am__append_132 = term-utils/write.1
-@BUILD_WRITE_TRUE@@MAKEINSTALL_DO_CHOWN_TRUE@@USE_TTY_GROUP_TRUE@am__append_133 = install-exec-hook-write
+# @BUILD_WALL_TRUE@am__append_128 = wall
+# @BUILD_WALL_TRUE@am__append_129 = term-utils/wall.1
+# @BUILD_WALL_TRUE@@MAKEINSTALL_DO_CHOWN_TRUE@@USE_TTY_GROUP_TRUE@am__append_130 = install-exec-hook-wall
+# @BUILD_WRITE_TRUE@am__append_131 = write
+# @BUILD_WRITE_TRUE@am__append_132 = term-utils/write.1
+# @BUILD_WRITE_TRUE@@MAKEINSTALL_DO_CHOWN_TRUE@@USE_TTY_GROUP_TRUE@am__append_133 = install-exec-hook-write
 @BUILD_LAST_TRUE@am__append_134 = last
 @BUILD_LAST_TRUE@am__append_135 = \
 @BUILD_LAST_TRUE@	login-utils/last.1 \
